src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+import React, { useState } from 'react';
+import i18n from '@dhis2/d2-i18n';
+import Paper from '@material-ui/core/Paper/Paper';
+import withStyles from '@material-ui/core/styles/withStyles';
+import { SingleSelect, SingleSelectOption } from '@dhis2/ui-core';
+import { LockedSelector } from '../../LockedSelector';
+import type { Props } from './SearchPage.types';
+import { Section, SectionHeaderSimple } from '../../Section';
+
+const getStyles = (theme: Theme) => ({
+    container: {
+        padding: '10px 24px 24px 24px',
+    },
+    paper: {
+    // marginBottom: theme.typography.pxToRem(10),
+        padding: theme.typography.pxToRem(10),
+        marginBottom: 2000,
+
+    },
+    customEmpty: {
+        textAlign: 'center',
+        padding: '8px 24px',
+    },
+    groupTitle: {
+        padding: '16px 12px',
+        fontWeight: 500,
+        fontSize: 16,
+    },
+    searchDomainSelectorSection: {
+        maxWidth: theme.typography.pxToRem(900),
+        marginBottom: theme.typography.pxToRem(20),
+    },
+    searchRow: {
+        display: 'flex',
+        padding: '8px 0',
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+    searchRowTitle: {
+        flexBasis: 200,
+        marginLeft: 16,
+    },
+    searchRowSelectElement: {
+        width: '100%',
+        marginRight: 8,
+    },
+});
+
+
+const Index = ({ classes, trackedEntityTypesWithCorrelatedPrograms, preselectedProgram }: Props) => {
+    const [selectedOption, choseSelected] = useState(preselectedProgram);
+
+    return (<>
+        <LockedSelector />
+        <div className={classes.container}>
+            <Paper className={classes.paper}>
+                <Section
+                    className={classes.searchDomainSelectorSection}
+                    header={
+                        <SectionHeaderSimple
+                            containerStyle={{ paddingLeft: 8, borderBottom: '1px solid #ECEFF1' }}
+                            title={i18n.t('Search')}
+                        />
+                    }
+                >
+                    <div className={classes.searchRow}>
+                        <div className={classes.searchRowTitle}>Search for</div>
+                        <div className={classes.searchRowSelectElement}>
+                            <SingleSelect
+                                onChange={({ selected }) => { choseSelected(selected); }}
+                                selected={selectedOption}
+                                empty={<div className={classes.customEmpty}>Custom empty component</div>}
```

Comment:
Since there is no optimization inside the SingleSelect component I guess it's ok to add the jsx inline like this for now.

Reply:
So that is not a blocker for you right?